server/src/main/resources/org/opensearch/tasks/task-index-mapping.json

```diff
@@ -34,6 +34,9 @@
           "start_time_in_millis": {
             "type": "long"
           },
+          "cancellation_time_millis": {
+            "type": "long"
+          },
```

Comment:
When changing a mapping, you need to increment the `version` field at the top, so future interactions with the index can detect and update the changed mappings.

Reply:
@dbwiddis I have updated the version from 4 to 5. If there are any other issues, I will address them immediately. Thank you for kindly letting me know! 😄